rudra/utils/github_utils.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Dharmendra G Patel <dhpatel@redhat.com>
+#
+"""Github utility providing functionality to read and write github resource."""
+from github import Github
+
+
+class GithubUtils:
+    """Class to perform action related to github like get content / raise PR etc."""
+
+    def __init__(self, github_token):
+        """Get github token to initialize the class."""
+        # Create github object using token
```

Comment:
These comments are redundant. Please remove these as the code itself is self explanatory. Comments are needed only the code does some magic which is hard to understand.

Ditto for all